src/shared/ListRepo/ReposTable/ReposTable.tsx

```diff
@@ -0,0 +1,212 @@
+    columns: getReposColumnsHelper({
+      inactive: repoDisplay === repoDisplayOptions.INACTIVE.text,
+      isCurrentUserPartOfOrg: isCurrentUserPartOfOrg,
+      owner,
+    }),
+    getCoreRowModel: getCoreRowModel(),
+    data: tableData,
+    state: {
+      sorting,
+    },
+    onSortingChange: setSorting,
+    getSortedRowModel: getSortedRowModel(),
+  })
+
+  if (!isFetching && isEmpty(tableData)) {
+    return <NoReposBlock searchValue={searchValue} />
+  }
+
+  return (
+    <div className="tableui">
+      <table>
+        <thead>
+          {table.getHeaderGroups().map((headerGroup) => (
+            <tr key={headerGroup.id}>
+              {headerGroup.headers.map((header) => (
+                <th
+                  key={header.id}
+                  colSpan={header.colSpan}
+                  scope="col"
+                  data-sortable={
+                    header.column.getCanSort() && header.column.id !== 'lines'
+                  }
+                  {...(header.column.id !== 'inactiveRepo' &&
+                  header.column.id !== 'lines'
+                    ? { onClick: header.column.getToggleSortingHandler() }
+                    : {})}
+                >
+                  <div
+                    className={`flex gap-1 ${
+                      header.id === 'coverage' ? 'flex-row-reverse' : ''
+                    } ${header.id === 'lines' ? 'justify-end' : ''}`}
+                  >
+                    {flexRender(
+                      header.column.columnDef.header,
+                      header.getContext()
+                    )}
+                    <span
+                      className="text-ds-blue-darker group-hover/columnheader:opacity-100"
+                      data-sort-direction={header.column.getIsSorted()}
+                    >
+                      <Icon name="arrowUp" size="sm" />
+                    </span>
+                  </div>
+                </th>
+              ))}
+            </tr>
+          ))}
+        </thead>
+        <tbody>
+          {table.getRowModel().rows.map((row) => (
+            <tr key={row.id}>
+              {row.getVisibleCells().map((cell) => (
+                <td
+                  key={cell.id}
+                  className={cs({
+                    'flex justify-end':
+                      cell.column.id === 'coverage' ||
+                      cell.column.id === 'inactiveRepo',
+                  })}
+                >
+                  {flexRender(cell.column.columnDef.cell, cell.getContext())}
+                </td>
+              ))}
+            </tr>
+          ))}
+        </tbody>
+      </table>
+      {hasNextPage && (
+        <div className="flex w-full justify-center">
+          <Button
+            hook="load-more"
+            isLoading={isFetchingNextPage}
+            onClick={fetchNextPage}
+            to={undefined}
+            disabled={false}
+          >
+            Load More
+          </Button>
+        </div>
+      )}
```

Comment:
Can you remove this and swap to using infinite scrolling? It's something we've been trying to move towards to enhance the users experience, you can see an example here: [load more trigger](https://github.com/codecov/gazebo/blob/main/src/pages/RepoPage/PullsTab/PullsTableTeam/PullsTableTeam.tsx#L28-L37), [loaders](https://github.com/codecov/gazebo/blob/main/src/pages/RepoPage/PullsTab/PullsTableTeam/PullsTableTeam.tsx#L178-L179).